datafusion/row/src/writer.rs

```diff
@@ -269,29 +269,29 @@ impl RowWriter {
 
 /// Stitch attributes of tuple in `batch` at `row_idx` and returns the tuple width
 pub fn write_row(
-    row: &mut RowWriter,
+    row_writer: &mut RowWriter,
```

Comment:
This renaming is unrelated, I think?

Reply:
I thought its better naming, because when reading the code I supposed the `row`  is either incoming row or row buffer, both  are not correct. row_writer better reflects the object imho.